src/pages/shared/components/SelectGasMenu/SelectGasMenuBody.tsx

```diff
@@ -62,8 +65,7 @@ const SelectGasMenuBody: FC<GasMenuProps> = ({
 			<StyledNumericInput
 				value={customGasPrice}
 				onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-					const newPrice = Number(e.target.value);
-					setCustomGasPrice(newPrice);
+					setCustomGasPrice(e.target.value);
```

Comment:
The reason why we had `0` is because `Number('')` evaluates to `0`. By using the input as a string we can now delete the whole input.